packages/engine/src/simulation/package/mod.rs

```diff
@@ -1,20 +1,178 @@
-/*!
-The central component of the HASH simulation engine is the Package system.
-
-The only presumption that the engine makes about a simulation project is that it creates
-a set of agents backed by the Datastore, which may have their state changed as
-simulation time progresses.
-
-Using only this presumption clearly does not provide enough utility to a simulation
-so packages can be added on top of that to create a simulation engine which provides
-the user with a wide array of functionality.
-
-For example, if the `BehaviorExecution` State Package is enabled, then the engine
-will execute behaviors on agents, depending on the behavior lists of the agents.
-
-A default collection of packages are usually used for the engine (
-see [`PackageConfig`](crate::simulation::config::PackageConfig)).
- */
+//! The central component of the HASH simulation engine is the Package system.
+//!
+//! The only presumption that the engine makes about a simulation project is that it creates
+//! a set of agents backed by the Datastore, which may have their state changed as
+//! simulation time progresses.
```

Comment:
```suggestion
```suggestion
//! The only invariant that the engine expects of a simulation project is that it creates
//! a set of agents backed by the Datastore, which may have their state changed as
//! simulation time progresses.
```